pb/proof.proto

```diff
@@ -0,0 +1,11 @@
+syntax="proto3";
+
+package proof.pb;
+
+message Proof {
+  int64 start     = 1;
+  int64 end       = 2;
+  repeated bytes nodes = 3;
+  bytes leafHash = 4;
+  bool isMaxNamespaceIDIgnored=5;
```

Comment:
[not blocking] Related: https://github.com/celestiaorg/nmt/issues/206 . Once we make this a protobuf field, I wonder if renaming it would be breaking. I think renaming shouldn't block this PR but wanted to call it out. cc: @staheri14

Reply:
Thanks @rootulp for this suggestion. I also agree that in line with https://github.com/celestiaorg/nmt/issues/206, it would be good to change this message field to `isMaxNamespaceIgnored`. I don't think it has to match the [same field in the `Proof` struct](https://github.com/celestiaorg/nmt/blob/3693c9a626b7ddaae996b3c75602f5accfa45739/proof.go#L48), so it should be fine to use `isMaxNamespaceIgnored` instead.